unicodetools/data/ucd/dev/emoji/ReadMe.txt

```diff
@@ -1,22 +1,22 @@
 # Unicode Character Database
-# Date: 2022-08-29, 23:20:20 GMT
-# © 2022 Unicode®, Inc.
+# Date: 2023-01-30
+# © 2023 Unicode®, Inc.
 # Unicode and the Unicode Logo are registered trademarks of Unicode, Inc. in the U.S. and other countries.
 # For terms of use, see https://www.unicode.org/terms_of_use.html
 
-This directory contains final data files for Unicode, Version 15.0
+This directory contains draft data files for Unicode, Version 15.1
 
-Public/15.0.0/emoji/
+Public/draft/UCD/ucd/emoji/
```

Comment:
Ditto. Although I know we've discussed the location of emoji data and how it's a strange case given certain data that's in UCD vs. stored separately.